layers/+completion/auto-completion/packages.el

```diff
@@ -109,11 +109,15 @@
 
       (add-hook 'company-completion-started-hook 'company-turn-off-fci)
       (add-hook 'company-completion-finished-hook 'company-maybe-turn-on-fci)
-      (add-hook 'company-completion-cancelled-hook 'company-maybe-turn-on-fci))
+      (add-hook 'company-completion-cancelled-hook 'company-maybe-turn-on-fci)
+
+      (add-hook 'text-mode-hook 'company-mode nil t))
     :config
     (progn
       (spacemacs|diminish company-mode " ⓐ" " a")
 
+      (spacemacs|add-company-backends :backends company-dabbrev
```

Comment:
This [list of company backends](https://github.com/syl20bnr/spacemacs/blob/cabe650f008b07f240d3a220960e2c84c5ea9918/layers/%2Bcompletion/auto-completion/config.el#L26-L32) is always enabled by default when calling [`spacemacs|add-company-backends`](https://github.com/syl20bnr/spacemacs/blob/cabe650f008b07f240d3a220960e2c84c5ea9918/layers/%2Bcompletion/auto-completion/funcs.el#L101-L103).

So you can just write
```elisp
(spacemacs|add-company-backends :modes text-mode)
```

Reply:
Updated